CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.  The format
 Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## 0.5.4 07-06-2021
+### Changed
+- Clear original export fields in the aggregation process. (#220, @stati)
+- Move bytes.buffer to []byte for set and message and optimize the way of encoding record, set,
```

Comment:
Move bytes.buffer to []byte for set and message. Optimize the encoding of record, set and message entities.

Reply:
Done. Thanks!